src/ca_biositing/pipeline/ca_biositing/pipeline/utils/reviewed_api_mappings.py

```
@@ -30,20 +57,1356 @@
+    'MILK  CONDENSED  WHOLE  SWEETENED  BULK': 'MILK',
+    'MILK  CONDENSED  WHOLE  UNSWEETENED  BULK': 'MILK',
+    'MILK  DRY  SKIM  ANIMAL': 'MILK',
+    'MILK  DRY  WHOLE': 'MILK',
+    'MILK  DRY  WHOLE  PACKAGE 5+LBS.': 'MILK',
+    'MILK  SKIM  EVAPORATED  CASE': 'MILK',
+    'MILK  WHOLE  EVAP & SWEETENED COND CASE': 'MILK',
+    'MILK  WHOLE  EVAPORATED & CONDENSED  CASE': 'MILK',
+    'MILLET (PROSO)': 'MILLET',
+    'MOHAIR': 'MOHAIR',
+    'MUSTARD': 'MUSTARD',
+    'MUSTARD': 'MUSTARD',
```

Ruff (0.15.5)

[error] Dictionary key literal `'MUSTARD'` repeated

Remove repeated key literal `'MUSTARD'`

(F601)